pkg/virt-controller/services/template.go

```diff
@@ -1385,9 +1386,20 @@ func getVirtiofsCapabilities() []k8sv1.Capability {
 	}
 }
 
+func requireDHCP(vmi *v1.VirtualMachineInstance) bool {
+	for _, iface := range vmi.Spec.Domain.Devices.Interfaces {
+		if iface.Bridge != nil || iface.Masquerade != nil || iface.Slirp != nil {
```

Comment:
I understand the need for bridge / masquerade.

Is it libvirt running this dhcp server for the virtual network ? (I'm not entirely familiar with slirp).

Reply:
It has nothing to do with DHCP. I moved it out of this function.

Reply:
Thanks, I think it is no longer misleading.

Just my curiosity here; what on slirp is requiring this capability ?

Reply:
Afaik we want to "host forward" all possible ports with slirp. That also includes well-known ports and therefore we need this capability with slirp.